pulpcore/pulpcore/app/models/task.py

```diff
@@ -274,23 +274,7 @@ class Task(Model):
         parent (models.ForeignKey): Task that spawned this task (if any)
         worker (models.ForeignKey): The worker that this task is in
     """
-
-    WAITING = 'waiting'
-    SKIPPED = 'skipped'
-    RUNNING = 'running'
-    COMPLETED = 'completed'
-    FAILED = 'failed'
-    CANCELED = 'canceled'
-    STATES = (
-        (WAITING, 'Waiting'),
-        (SKIPPED, 'Skipped'),
-        (RUNNING, 'Running'),
-        (COMPLETED, 'Completed'),
-        (FAILED, 'Failed'),
-        (CANCELED, 'Canceled')
-    )
-
-    state = models.TextField(choices=STATES)
+    state = models.TextField(choices=TASK_CHOICES, default=TASK_STATES.WAITING)
```

Comment:
This is the only change I am a little unsure about.  ProgressReport does set the default to "waiting", and it is a reasonable default IMO.  Better than setting no state at all, personally.

Reply:
This might be reasonable, but lets restrict this change to DRYing up only.